frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/LockoutInformation.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.di.authentication.frontendapi.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import uk.gov.di.authentication.shared.entity.JourneyType;
+import uk.gov.di.authentication.shared.entity.MFAMethodType;
+
+public class LockoutInformation {
```

Comment:
We can make this a record and reduce some of the boilerplate - https://github.com/govuk-one-login/authentication-api/blob/main/frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/AccountInterventionsResponse.java